js/custom.js

```diff
@@ -1,38 +1,71 @@
-(function() {
+(function () {
   var burger = document.querySelector(".burger");
   var menu = document.querySelector("#" + burger.dataset.target);
-  burger.addEventListener("click", function() {
+  burger.addEventListener("click", function () {
     burger.classList.toggle("is-active");
     menu.classList.toggle("is-active");
   });
 })();
 
-document.querySelectorAll("#nav li").forEach(function(navEl) {
-  navEl.onclick = function() {
+var ARROW_UP_KEY = 38;
+var ARROW_DOWN_KEY = 40;
+var ARROW_RIGHT_KEY = 39;
+var ARROW_LEFT_KEY = 37;
+
+document.querySelectorAll("#nav li").forEach(function (navEl, _, parent) {
```

Comment:
I'd recommend you using delegation (event bubbling) instead of handling element's events separately.

Reply:
I'll fix it in the next task.